torch/nn/_functions/rnn.py

```diff
@@ -7,19 +7,22 @@
     pass
 
 
-def RNNReLUCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
-    hy = F.relu(F.linear(input, w_ih, b_ih) + F.linear(hidden, w_hh, b_hh))
+def RNNReLUCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None, skip_input=False):
```

Comment:
Actually, wouldn't it be better to say that `skip_input is True` when `w_ih is None`?